gateway/1.7/modules/ROOT/pages/flex-troubleshoot-platform-connection.adoc

```diff
@@ -0,0 +1,51 @@
+= Troubleshooting Platform Connection
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+
+When configuring Flex Gateway in either mode, it is possible for the Flex Replica's `registration.yaml` to not be present or for the replica to lose connection to Anypoint Platform.
+
+To troubleshoot if a registration file is present and if Flex Gateway can make platform connections, Flex Gateway provides the following command:
+
+[source,ssh]
+----
+flexctl check connections
+----
+
+The `flexctl check connections` first ensures a `registration.yaml` file is present, and then checks if the replica can connect to the URLs defined in the registration file. If the command is successful, it returns the following `info` message:
+
+----
+$ flectl check connections
+[flexctl][info] Connection to platform endpoints successful.
+----
+
+== No Registration File Found
+
+If the `registration.yaml` file is not found, Flex Gateway returns the following error message:
+
+----
+$ flexctl check connections
+Error: no registration config
+----
+
+To troubleshoot an absent registration file, check your Flex Replica configuration folder. To learn more about registering a Flex Gateway, see following section for your relevant mode:
+
+* xref:flex-conn-reg-run.adoc[]
+* xref:flex-local-reg-run.adoc[]
+
+== Failed Connection
+
+There are multiple reasons Flex Gateway may not be able to connect to a platform endpoint, such as an error in the gateway's environment or a network issue. How you troubleshoot your error is dependent on the error returned.
+
+If Flex Gateway cannot to a specific URL, it returns an error message that includes the URL that was unreachable and the error message returned from the connection attempt. The error message appears similar to the following:
```

Comment:
1. missing word `If Flex Gateway cannot **connect** to a specific URL`
2. I'm not sure about `The error message appears similar to the following`, since it may not really look like that. I would say something like `For example, this is an error shown when the platform is unreachable through the gateway because of no internet connection`